### PR TITLE
GET data and save it in an array

### PR DESCRIPTION
**Pull request by IbrahimLama**

### CODE REVIEW

File: src/containers/Contact/index.js [hunk 1]
```diff
@@ -50,9 +51,20 @@ class Contact extends React.Component{
...
     };
   }
-  
+
+  componentDidMount()
+  {
+    axios.get(`https://gorest.co.in/public-api/users`).then(res =>
+      {
+        console.log(res);
+        this.setState({conatctList:res.data});
```
Comment (IbrahimLama) [inline src/containers/Contact/index.js:63]: reformat to 
```
this.setState({
  conatctList: res.data
});

File: src/components/AboutImg/index.js [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
-                {props.career}
-            </p>
-            <p className="Body__Parag">
-                {props.parag}
+                {props.email}
             </p>
             <button className="ImgInfo__Btn"
-                    type="button">Contact</button>  
+                    type="submit"
+                    disabled={props.status.length>6}
```
Comment (IbrahimLama) [inline src/components/AboutImg/index.js:20]: its not the right way to check it, 
check if status equal "active"

File: src/containers/About/index.js [hunk 3]
```diff
@@ -12,33 +13,67 @@ class About extends React.Component{
...
+                },{
+                src:johnImage,
+                },{
+                src:johnImage,
+                },{
+                src:johnImage,
+                },{
+                src:johnImage,
+                },{
+                src:johnImage,
```
Comment (IbrahimLama) [inline src/containers/About/index.js:57]: no need for this array, make the image static inside the component

File: package.json [hunk 4]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.21.0",
     "formik": "^2.2.1",
+    "jquery": "^3.5.1",
```
Comment (IbrahimLama) [inline package.json:11]: why you added jquery?

File: src/components/AboutImg/index.js [hunk 5]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
 
 function AboutImg(props){
-     return(
+    return(
         <div className="AboutInner__ImgInfo"> 
             <img className="ImgInfo__Img"
-                 src={props.src} 
-                 alt={props.alt}
+                src={props.src}
+                alt={props.alt}
```
Comment (IbrahimLama) [inline src/components/AboutImg/index.js:9]: fix indentation

File: src/components/AboutImg/index.js [hunk 6]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
-            </p>
-            <button className="ImgInfo__Btn"
-                    type="button">Contact</button>  
+            <form action={`mailto:${props.email} }`}>
+                <input className="ImgInfo__Btn"
+                       type="submit" 
+                       disabled={props.status=="Inactive"}
+                       value="Contact" 
+                />
+            </form>
```
Comment (IbrahimLama) [inline src/components/AboutImg/index.js:23]: mailto can be used easily with anchor tag

**Comment by amonamoqadi** [comment]
Yes i made it, but i faced a problem with disabling it
So i used this

File: src/components/AboutImg/index.js [hunk 7]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
             </p>
-            <p className="Body__Parag">
-                {props.parag}
-            </p>
-            <button className="ImgInfo__Btn"
-                    type="button">Contact</button>  
+            <form action={`mailto:${props.email} }`}>
+                <input className="ImgInfo__Btn"
+                       type="submit" 
+                       disabled={props.status=="Inactive"}
```
Comment (IbrahimLama) [inline src/components/AboutImg/index.js:20]: read about and use triple equal

**Comment by amonamoqadi** [comment]
I used it when i was working on disabling href tag<a>

File: package.json (hunk 4 above)
Comment (IbrahimLama) [reply package.json:11]: its not preferred to use jquery with react, if it is not used now uninstall it

File: src/components/AboutImg/index.js (hunk 6 above)
Comment (IbrahimLama) [reply src/components/AboutImg/index.js:23]: check this https://www.rapidtables.com/web/html/mailto.html

**Comment by amonamoqadi** [comment]
Yup it's not used

File: src/components/AboutImg/index.js [hunk 8]
```diff
@@ -2,23 +2,29 @@ import React from 'react';
 import "./style.scss";
 
 function AboutImg(props){
-     return(
+    const STATUS= {INACTIVE:"Inactive",ACTIVE:"Active"}
+    var isDisabled= props.status===STATUS.INACTIVE;
```
Comment (IbrahimLama) [inline src/components/AboutImg/index.js:6]: reformat to 
```
const STATUS = { 
     INACTIVE: "Inactive",
      ACTIVE: "Active"
}
const isDisabled = props.status === STATUS.INACTIVE;

**Pull request merged**